src/main/resources/db/migration/media_table.sql

```diff
@@ -0,0 +1,22 @@
+-- Table: darts_schema.notification
+
+-- DROP TABLE IF EXISTS darts_schema.media;
+
+CREATE TABLE IF NOT EXISTS darts_schema.media
```

Comment:
One for @hemantasharma1129 / @jackmaloney to confirm the ask...
The entity we have under `common` was an initial mapping based on what Barry had in Sharepoint (v5).

It looks like there is a newer version (v8) where the DDL is

The common entities would need a refresh. e.g. add sequences,remove character/numeric size limits, change DATE to TIMESTAMP
```
CREATE TABLE DARTS.moj_media
(moj_med_id                 INTEGER					NOT NULL
,moj_crt_id                 INTEGER
,r_media_object_id          CHARACTER VARYING(16)
,c_channel                  NUMERIC
,c_total_channels           NUMERIC
,c_reference_id             CHARACTER VARYING
,c_start                    TIMESTAMP
,c_end                      TIMESTAMP
,c_courtroom                CHARACTER VARYING
,c_reporting_restrictions   NUMERIC
,c_case_id                  CHARACTER VARYING(32)[]                    --this is a placeholder for moj_case_document_r.c_case_id, known to be repeated for moj_media object types
,r_case_object_id           CHARACTER VARYING(16)[]                    --this is a placeholder for moj_case_s.r_object_id, and must be derived from moj_case_document_r.c_case_id
,r_version_label            CHARACTER VARYING(32)
,i_superseded               BOOLEAN
,i_version_label            SMALLINT);
```